os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/i2s_api.c

```diff
@@ -378,7 +378,7 @@ void i2s_set_direction(i2s_t *obj, int trx_type)
   */
 void i2s_set_param(i2s_t *obj, int channel_num, int rate, int word_len)
 {
-	uint32_t clock_mode;
+	uint32_t clock_mode = 0;
```

Comment:
Is this a build warning fix? Not svace issue?

Reply:
Yes, this is a build warning.

CC:  component/mbed/targets/hal/rtl8730e/i2s_api.c
component/mbed/targets/hal/rtl8730e/i2s_api.c: In function 'i2s_set_param':
component/mbed/targets/hal/rtl8730e/i2s_api.c:421:40: warning: 'clock_mode' may be used uninitialized in this function [-Wmaybe-uninitialized]
  SP_InitStruct[obj->i2s_idx].SP_SelClk = clock_mode;
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~